packages/api/models/Project.js

```diff
@@ -3,6 +3,7 @@ import { dynamoDbDocumentClient } from '../dynamodb-init'
 
 const ProjectTable = new Table({
   name: process.env.PROJECT_TABLE,
+  // userId#programId
```

Comment:
remove

Reply:
I'll update this to something like `// NOTE: The partition key looks like userId#programId`